pkg/http/api.go

```diff
@@ -615,7 +615,7 @@ func (a *api) onDirectMessage(reqCtx *fasthttp.RequestCtx) {
 func (a *api) onCreateActorReminder(reqCtx *fasthttp.RequestCtx) {
 	if a.actor == nil {
 		msg := NewErrorResponse("ERR_ACTOR_RUNTIME_NOT_FOUND", messages.ErrActorRuntimeNotFound)
-		respondWithError(reqCtx, fasthttp.StatusBadRequest, msg)
+		respondWithError(reqCtx, fasthttp.StatusInternalServerError, msg)
```

Comment:
What exactly does `ERR_ACTOR_RUNTIME_NOT_FOUND` mean? Does it mean that the actor runtime hasn't been configured with a compatible state store?

Reply:
I can mean that, or it can mean that the actor subsystem failed to initialize for whatever reason (placement service missing?).

Reply:
OK 500 for those cases make sense.